def handle_record_not_found_error(record_name: str) -> str:
    return "You don't have a {record_name} with given `id`"
